.circleci/config.yml

```diff
@@ -105,7 +106,7 @@ executors:
       # Set cluster workers to 5 (5 cluster nodes)
       # NUM_NODES represent number of parallel test executions
       # NUM_WORKERS represent number of nodes in a k8 cluster
-      NUM_NODES: 5 
+      NUM_NODES: 3 
```

Comment:
what is this change for?

Reply:
All these are part of the merge, and no manual changes are involved. :-). 
This change came from Param, a couple of days ago as part of a PR.

Reply:
@pdhanoyasplunk any specific reason for this change?

Reply:
@gaurav-splunk `NUM_NODES` is for parallel execution. Having more parallel test have no benefit and are causing flakiness in test. Hence I reduced the number of parallel test for more reliability in nightly runs.